BSP/Simulator/Hardware/Contactor.py

```diff
@@ -0,0 +1,27 @@
+import csv
+import os
+import fcntl
+
+
+file = "BSP/Simulator/Hardware/Data/Contactors.csv"
+
+#Reads file and returns 1 or 0, 1 is on, 0 is off
+def Read_Contactor():
```

Comment:
Naming convention. Each of the modules has been calling their respective reading functions just `read()`. Easier to call in the simulator.py file